src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,61 @@
+package racing.domain;
+
+public class Car implements Comparable<Car> {
+
+    private static final int MAX_NAME_LENGTH = 5;
+    public static final int MIN_FORWARD_CONDITION = 4;
+    public static final String POSITION_MARKER = "-";
+
+    private final String name;
+    private int position;
+
+    public Car(String name) {
+        validate(name);
+        this.name = name;
+        this.position = 0;
+    }
```

Comment:
생성자가 여러개 있다면 다음과 같이 관리해볼 수도 있어요.
```suggestion
    public Car(String name) {
        this(name, 0);
    }
```